package-lock.json

```diff
@@ -20768,7 +20768,7 @@
     },
     "src/JSInterop/Microsoft.JSInterop.JS/src": {
       "name": "@microsoft/dotnet-js-interop",
-      "version": "8.0.0-dev",
+      "version": "9.0.0-dev",
```

Comment:
Please undo this change.
Or is it related to this PR?

Reply:
PR has been updated.
No it was not related